smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/ProtocolGenerator.java

```diff
@@ -148,6 +148,13 @@ default void generateSharedComponents(GenerationContext context) {
      */
     void generateResponseSerializers(GenerationContext context);
 
+    /**
+     * Generates the code used to serialize unmodeled exceptions for servers.
+     *
+     * @param serverContext Serialization context.
+     */
+    void generateFrameworkExceptionSerializer(GenerationContext serverContext);
```

Comment:
We use the term "Error" in other interfaces so we should here as well for consistency

Reply:
I was actually halfway to changing Exception to Error globally in my review yesterday before submitting it but then I saw how often I was handling `SmithyException` and `SerializationException` and `UnknownOperationException` and I left it. I can change the method names in the generator for sure.